### PR TITLE
Added using Sylius components instructions

### PR DESCRIPTION
**Pull request by michalmarcinkowski**
| Q             | A
| ------------- | ---
| Doc fix?      | yes
| New docs?     | yes
| Fixed tickets |

### CODE REVIEW

File: components/index.rst [hunk 1]
```diff
@@ -1,12 +1,19 @@
-PHP Ecommerce Components
-========================
+PHP E-Commerce Components
```
Comment (pamil) [inline components/index.rst]: IMHO we should name it `PHP Components` and do the same with bundles. We already have Grid component, ResourceBundle, ThemeBundle, MetadataBundle, UserBudle - they don't have anything in common with E-Commerce.

Comment (michalmarcinkowski) [inline components/index.rst]: I think it make sense, but we should prepare separate PR with this change.

Comment (pjedrzejewski) [inline components/index.rst]: :+1:

File: components/using_components.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    $taxRate->setAmount(0.23);
+
+    $tax = $calculator->calculate(100, $taxRate);
+
+    // ...
+
+Using all of the Components
+---------------------------
+
+If you want to use all of the Symfony Components, then instead of adding
```
Comment (pjedrzejewski) [inline components/using_components.rst]: Sylius :P

**Comment by pjedrzejewski** [comment]
:+1:

**Pull request merged**